src/qml/controls/OptionSwitch.qml

```diff
@@ -4,14 +4,15 @@
 
 import QtQuick 2.15
 import QtQuick.Controls 2.15
+import QtGraphicalEffects 1.15
 
 Switch {
     id: root
     indicator: Rectangle {
         implicitWidth: 45
         implicitHeight: 28
         x: root.leftPadding
-        y: parent.height / 2 - height / 2
+        y: Math.round((parent.height - height) / 2)
```

Comment:
Follow ups to #101 should be in their own PR with well structured and focused commits.

Reply:
Addressed in 109682b971e08bf282b82b7362c136f68340abee